components/ListOperators/register.jsx

```diff
@@ -13,9 +13,9 @@ const RegisterOperators = () => {
 
   return (
     <>
-      <Title level={2}>Register Operator</Title>
+      <Title level={2}>Mint Operator</Title>
```

Comment:
I am unsure, but if this refers to register an operator on an already minted state, it should be 'register', no?